CONTRIBUTING.rst

```diff
@@ -92,45 +92,84 @@ Pull Request Guidelines
 
 Before you submit a pull request, check that it meets these guidelines:
 
-1. The pull request should include tests (see notes below).
+1. The pull request should include unit tests (see notes below).
 2. If the pull request adds functionality, the docs should be updated. Put
-   your new functionality into a function with a docstring, and add the
-   feature to the list in README.rst.
-3. The pull request should work for Python 2.6, 2.7, and for PyPy. Check
+   your new functionality into a function with a docstring, and make sure
+   your feature is reflected in one of the `.rst` files under `./docs/`.
+   If an existing module exists and your feature is appended to it,
+   then you can ignore this step as Sphinx will render it automatically.
+   Otherwise, create a new `.rst` file.
+3. **(Optional)** Create a Jupyter Notebook or a detailed documentation `.rst`
+   under `./docs/examples/` to explain exactly how the feature should be used.
+4. The pull request should work for Python 2.6, 2.7, 3.6, 3.7, and for PyPy. Check
```

Comment:
As long as we are changing this line, we currently test with 2.7, 3.5, 3.6, and 3.7.  Let's remove 2.6 and PyPy and add 3.5 to the list.